### ISSUE TITLE
Empty date is still read by screen readers

### ISSUE THREAD

**Issue by rowanmanning** [Financial-Times/o-date#105]
Reported by @alicebartlett: When the requested date format returns an empty string, the `time` element is not visible because it has no content. However a screen reader (we tested using VoiceOver) will still read the date because the `time` element has a `title` attribute.

##  What

We think that the experience for a person using assistive tech should be the same as for a person who is not. I'm proposing that if the date string in [this method](https://github.com/Financial-Times/o-date/blob/master/main.js#L39) is an empty string, we add an `aria-hidden` attribute to the `time` element. This will mean that the date is both not visible, and not read.
